docs/tutorials/analysis/time/light_curve_simulation.ipynb

```diff
@@ -431,7 +431,25 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.0"
+   "version": "3.7.3"
+  },
```

Comment:
Can you check your version of `nbconvert`? Like:
```
import nbconvert
print(nbconvert.__version__)
```
I think in more recent versions this meta data is not present anymore...

Reply:
ok upgraded to 6.1.0